### DISCUSSION TITLE
Fix RigTools EdPuzzle Script

### DISCUSSION THREAD

**Discussion by gambitin** [3kh0/ext-remover#1264] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
The script is outdated, it uses the old gui instead of the new one.

**Comment by S-PScripts** [comment]
3kh0 can't fix it, anyways can't you use that javascript executor in rigtools or something

**Reply by gambitin** [reply]
Dunno how to, im austitic

**Reply by S-PScripts** [reply]
just get the javascript for the edpuzzle script and paste it in there

**Reply by buddyted** [reply]
what is that I need that also

**Reply by gambitin** [reply]
i figured it out lol, just had to click run on dev tools and then run on the tab

**Reply by S-PScripts** [reply]
ok
